Code/app_vars.py

```diff
@@ -37,6 +37,7 @@ def init(cls, debug=False) -> None:
         cls._user_data_path.mkdir(parents=True, exist_ok=True)
         cls._load_user_config()
         cls.set("debug", debug)
+        cls.get_mods_path()
```

Comment:
I think I should move this to path validation, rather than installing it upon program startup.

Reply:
Even where the original path was removed

Reply:
Whatever you think is best.
I just want to point out the critical issues.

Reply:
Anyway, it works now.

Reply:
good